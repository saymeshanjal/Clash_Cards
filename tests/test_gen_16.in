21
12 208 4
14 209 4
4 189 5
10 207 3
2 247 4
12 99 5
10 135 2
12 110 3
6 24 5
16 55 3
12 46 5
16 4 5
6 70 1
10 106 5
6 0 4
4 236 5
6 124 3
16 27 1
16 212 1
4 21 4
10 185 4