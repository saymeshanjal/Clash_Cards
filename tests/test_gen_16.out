208 441 275
910 1520 1002
0 6 0
110 142 0
0 2 0
219 705 0
76 77 99
430 494 0
1 19 0
3763 9107 0
170 754 0
0 12870 0
10 10 0
45 207 0
0 20 0
0 6 0
6 14 0
6005 6865 0
6435 6435 0
0 5 1
44 133 75
